translations/de/hints/bryukh.html

```diff
@@ -1,40 +1,43 @@
 <body data-author="bryukh">
 <div class="question-answer">
-    <div class="question">
-        I have no idea how to start solving this mission
-    </div>
-    <div class="answer">
-        To iterate indexes in an array use "<a href="https://docs.python.org/2/library/functions.html#range">range</a>".
-<pre class="brush: python">
+  <div class="question">
+    Ich habe keine Ahnung wo ich mit der Lösung anfangen soll.
```

Comment:
Hallo. Mir gefällt dieser Satz gar nicht. (Und täglich grüsst das Murmeltier, hatten wir nicht schon mal diesselbe Diskussion? 🤔). "...wo ich mit der Lösung anfangen soll." sagt doch das hier schon eine Lösung existiert, aber man weiss nicht wo mit dieser Lösung zuerst ansetzen.
Wie wäre es mit "...wie ich anfangen soll" plain and simple.